<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoiConfigWidget</class>
 <widget class="QWidget" name="RoiConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>CAM3裁剪框像素坐标</string>
   </property>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam3_x_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam3_height_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam3_width_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>宽度：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>高度：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam3_y_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>CAM4裁剪框像素坐标</string>
   </property>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam4_x_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam4_height_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_58">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam4_width_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>宽度：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>高度：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="cam4_y_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
